tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -431,8 +431,14 @@ Verify That MT-SRE Are Not Paged For Alerts In Clusters Used For Development Or
 Alerts Suite Setup
     [Documentation]    Test suite configuration
     Set Library Search Order    SeleniumLibrary
+    Skip If RHODS Is Self-Managed
     RHOSi Setup
 
+Alerts Suite Teardown
+    [Documentation]    Test suite teardown
+    Skip If RHODS Is Self-Managed
+    RHOSi Teardown
```

Comment:
probably not needed, since we have the skip in the `Suite Availability Teardown` directly as per PR #671

Reply:
I'll fix that in a future PR if needed

Reply:
ok